vis/xAPISG-CompletablesTimes.ipynb

```diff
@@ -59,33 +68,72 @@
     "                        maximum = total_time\n",
     "                    if total_time < minimum:\n",
     "                        minimum = total_time\n",
+    "        if minimum==float('Inf'):\n",
+    "            minimum=maximum\n",
     "        max_times.append(maximum)\n",
     "        min_times.append(minimum)\n",
-    "        \n",
-    "        \n",
+    "    N=len(completablesSelectedCompletableTime)\n",
+    "    n_groups=N\n",
+    "    if eyex==0:  #getAllDataInAListOfList\n",
+    "        completablesSelectedCompletableTime=[completablesSelectedCompletableTime]\n",
+    "        min_times=[min_times]\n",
+    "        max_times=[max_times]\n",
+    "        nbtimes=1\n",
+    "    elif eyex==-1:  #getnbFirstData\n",
+    "        nb=10\n",
+    "        nbtimes=1\n",
+    "        completablesSelectedCompletableTime=[completablesSelectedCompletableTime[:nb]]\n",
+    "        max_times=[max_times[:nb]]\n",
+    "        min_times=[min_times[:nb]]\n",
+    "        if N>nb:\n",
+    "            N=nb\n",
+    "    elif eyex==-2:  #getnbLastData\n",
+    "        nb=10\n",
+    "        nbtimes=1\n",
```

Comment:
Some assignments could be performed outside if/elif/else for instance this nbtimes=1, which only changes in the last else